frontend/common/src/helpers/formUtils.ts

```diff
@@ -107,3 +107,15 @@ export function matchStringsCaseDiacriticInsensitive(
     matchStringCaseDiacriticInsensitive(needle, name),
   );
 }
+
+/**
+ * Returns the total number of words in a string.
+ * @param text String that you want to count the number of words.
+ * @returns number
+ */
+export const countNumberOfWords = (text: string): number => {
```

Comment:
Very nice... but I don't fully trust it without a test. :grin:  Could you add a few, please?

Reply:
I knew I was forgetting something 👍

Reply:
New tests look great!